package.json

```diff
@@ -82,6 +82,7 @@
     "string-replace-async": "^1.2.1",
     "style-loader": "^0.19.0",
     "tar": "^3.1.5",
+    "term.js": "0.0.7",
```

Comment:
This doesn't seem to be required anywhere.

Reply:
It's needed or blessed.terminal doesn't work